### PR TITLE
fix(user): avoid duplicated en_US language selection

### PR DESCRIPTION
**Pull request by victorvassilev**

### CODE REVIEW

File: features/bootstrap/LanguageSelectionContext.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        $this->currentPage = new ParametersMyAccountPage($this);
+        $this->currentPage->save();
+    }
+
+    /**
+     * @When I select the language dropdown
+     */
+    public function selectTheLanguageDropdown()
+    {
+        $this->visit('main.php?p=50104&o=c');
```
Comment (kduret) [inline features/bootstrap/LanguageSelectionContext.php]: you could instantiate ParametersMyAccountPage

File: features/bootstrap/LanguageSelectionContext.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        $this->assertFind('css', 'select[name="contact_lang"]');
+    }
+
+    /**
+     * @Then I can see unique human readable language options
+     */
+    public function iCanSeeProperDropdownListItems()
+    {
+        $this->visit('main.php?p=50104&o=c');
```
Comment (kduret) [inline features/bootstrap/LanguageSelectionContext.php]: you could instantiate ParametersMyAccountPage

File: features/bootstrap/LanguageSelectionContext.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        $enLang = $this->assertFind('css', 'select[name="contact_lang"] > option[value = "en_US.UTF-8"]');
+
+        if(is_array($enLang) || $enLang->getText() !== 'en_US') {
```
Comment (kduret) [inline features/bootstrap/LanguageSelectionContext.php]:
```suggestion
        if (is_array($enLang) || $enLang->getText() !== 'en_US') {
```

File: www/include/common/common-Func.php [hunk 4]
```diff
@@ -1805,19 +1805,22 @@ function getLangs()
...
                 if (is_dir("$chemintotal/$file") && strcmp($file, ".") && strcmp($file, "..")) {
-                    $langs[$file] = $file;
+                    $langTitle = str_replace('.UTF-8', '', $file);
+                    $langs[$file] = $langTitle;
                 }
             }
             closedir($handle);
         }
     }
+    if (!array_key_exists('en_US.UTF-8', $langs)){
```
Comment (kduret) [inline www/include/common/common-Func.php]:
```suggestion
    if (!array_key_exists('en_US.UTF-8', $langs)) {
```

File: features/bootstrap/LanguageSelectionContext.php (hunk 1 above)
Comment (victorvassilev) [reply features/bootstrap/LanguageSelectionContext.php]: changed as requested

File: features/bootstrap/LanguageSelectionContext.php (hunk 2 above)
Comment (victorvassilev) [reply features/bootstrap/LanguageSelectionContext.php]: changed as requested

**Pull request merged**